### ISSUE TITLE
Use "enums" or an object for more readability of Abilities map

### ISSUE THREAD

**Issue by seanohue** [KittyGarrison/evercraft_kata#2]
Right now the abilities.json uses an array to list the abilities. Arrays are possibly best for unordered collections where the key doesn't really matter. In this case, one has to remember that abilities[0] is Strength, abilities[1] is Dexterity, and so on.

Instead, consider an object where the keys are strings, making accessing the ability label a bit easier for developers:
```
{
  "STR": "Strength"
}
```

and so on.

In addition, it's also okay to have these constants in your JavaScript code rather than JSON. There's not really much of a performance advantage either way, so go with whichever seems easiest for you. That said, it's good that you learned that JSON is requirable,  and so on.

**Comment by seanohue** [comment]
Why is this a good refactor? This will make it so that if you have to add more data besides just the ability's label, you can make the value of "STR" into an object rather than a string:

```
{
  "STR": {
        "name": "Strength",
        "description: "How strong your character is, etc. etc."
        }
}
```

**Comment by seanohue** [comment]
I posted some comments on this commit:
https://github.com/KittyGarrison/evercraft_kata/commit/f4504f287f3f470777f76d1233c9f75958bf1b52

**Comment by seanohue** [comment]
Closed by 0831c8e67ba4f8adb800e4cfe2504a9233541d1f
